### DISCUSSION TITLE
Two independent screeners having screened using different ASReview versions (1.6.1 & 1.6.2)

### DISCUSSION THREAD

**Discussion by Morle2104** [asreview/asreview#1836] [Q&A] [answered]
Dear all, 

On our project, we have two independent screeners that work independently on the same dataset with the same prior knowledge and the same model settings. At the beginning of the set-up, the project was set up by one screener & then shared with the other screener to ensure that the set up is the exact same. However, now it was noticed that one screener has been working with version 1.6.1 and the other with version 1.6.2.  I am wondering in what way this will affect the comparison that we can draw between the two independently screened datasets, and whether this is a significant issue. What are your thoughts on this? Further, would you recommend that both screeners update their version now to the same version while they are still both in the screening phase (i.e., have not reached the stopping rule yet)? I tried to find a discussion on github about this topic, but I didn't find anything yet.

Many thanks in advance for your input!

**Comment by J535D165** [comment, marked as answer]
Thanks for the question. Short answer: nothing to worry about. You can go ahead with this. 

Longer answer: We use "Semver" for versioning. For example, when you see version numbers like 1.6.2, it tells you how the app has been updated:

- The first number is for major changes, like model updates and significant workflow changes.
- The second number is for minor changes, like new features. Old features will behave the same. 
- The third number is for patches or bug fixes.

https://github.com/asreview/asreview/releases/tag/v1.6.2 introduced a fix to make the import button work again for people using ASReview via Docker (most likely not you).

**Reply by Morle2104** [reply]
Oh great, many thanks for the quick reply! :-) Glad to hear that it's not a problem at all. And good to know the indexing of the versioning!

**Reply by Morle2104** [reply]
Hello. Just a quick follow-up to my initial question. Is it a problem if screeners during sequential screening (i.e., person A screens and then exports project file and person B imports this project file and continues screening in it) work with different versions, such as 1.6.1, 1.6.2 and 1.6.3, in the same dataset? If I understand the indexing above correctly, this should as well be no problem right? Many thanks in advance! :-)

**Reply by Rensvandeschoot** [reply]
indeed, same as was explained by @J535D165 :-)

**Comment by tayates** [comment]
@Morle2104 I am interested in how you are using ASReview with two reviewers. 

I had planned a standard systematic review, with me and a second reviewer screening a 10% sample of titles and abstracts then (assuming good agreement) me screening the remaining titles and abstracts alone (likely around 15k references). 

I am hoping to use ASReview to save me many weeks of work. But, with previous reviews, use of a second reviewer did flag some errors in screening. How are you integrating a second reviewer in your ASReview project?

Thanks!

**Reply by Morle2104** [reply]
Hi @tayates! I just started a new discussion here: https://github.com/asreview/asreview/discussions/1854#discussion-7211757
That may be interesting to you as well.

Regarding your question, we used option 1 (https://asreview.nl/blog/seven-ways-to-integrate-asreview/#multiple) for the double screening. And then, like in traditional screening, we compare our screening decisions for the abstracts that both screeners have seen (i.e., it can be that some are only seen by one screener) and resolve any disagreements.

**Reply by tayates** [reply]
Thanks. I quite like the look of option 6 - asking my second reviewer to work through studies I excluded, with ASReview showing him first those studies most likely to be relevant (i.e. those I most likely misclassified).

**Reply by Rensvandeschoot** [reply]
Nice solutions!
